torch/masked/maskedtensor/creation.py

```diff
@@ -15,9 +15,9 @@
 #     torch.as_tensor - differentiable constructor that preserves the autograd history
 
 
-def masked_tensor(data, mask, requires_grad=False):
+def masked_tensor(data: Any, mask: Any, requires_grad: bool = False) -> MaskedTensor:
```

Comment:
Instead of using `Any` consider using `object` (here and below)
```
def masked_tensor(data: object, mask: object, requires_grad: bool = False) -> MaskedTensor:
```

today there won't be a difference - because MaskedTensor's constructor is untyped. But when someone fixes MaskedTensor to take real types if these are `Any` then this will blindly continue to work. But if they're `object` then suddenly this function will (correctly) complain about the types being passed to `MaskedTensor` and then those fixes will continue to propagate up.